io.github.niccokunzmann.python_dhcp_server.yml

```diff
@@ -0,0 +1,87 @@
+id: io.github.niccokunzmann.python_dhcp_server
+# The runtime is the environment that the application will run in, so you can use dependencies it provides
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+# This is the command that will be run when the application is launched
+command: python_dhcp_server
+# These are the permissions that the application needs
+# Read more about finishing here: https://docs.flatpak.org/en/latest/manifests.html#finishing
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --allow=multiarch
+  - --persist=/app/config
+modules:
+  - name: python_dhcp_server
+    # There are other types of buildsystems, like autotools, cmake, cmake-ninja, simple, qmake
+    buildsystem: simple
+    # see https://docs.flatpak.org/en/latest/manifests.html#modules
+    build-commands:
+      - ./inside.sh
+    sources:
+      - type: dir
+        path: app
+    # for testing
+      # - type: dir
+      #   path: python_dhcp_server
+    # for release
+      - type: git
+        url: https://github.com/niccokunzmann/python_dhcp_server.git
+        tag: "0.8"
+        commit: "cffdfef4e22e2e2d8cbc45abe5f9560c51816bcb"
+      # - type: archive
+      #   url: https://github.com/niccokunzmann/python_dhcp_server/archive/refs/tags/0.8.tar.gz
+      #   # wget -O- 'https://github.com/niccokunzmann/python_dhcp_server/archive/refs/tags/0.8.tar.gz' | sha256sum
+      #   sha256: c489fb0e6d302170d11a009b601ca22073a6e42110a363dcb32560b2c3af61f9
+
+  # install tkinter, see https://discourse.flathub.org/t/how-can-i-package-a-python-tkinter-application/3795/4
+  # copied from https://github.com/flathub/net.scribus.Scribus/blob/master/net.scribus.Scribus.yaml
+  # The `tkinter` module is missing from the Freedesktop Sdk's Python installation.
+  - name: tkinter
```

Comment:
these two module should be above the "main" one. Dependencies come first.

Reply:
done!